File: articles/proxy-variables-and-measurement-error/prolog/proxy_measurement_risk.pl
measurement_risk(ValidityGap, Missingness, DifferentialError, LabelError, Score) :-
    Score is (ValidityGap + Missingness + DifferentialError + LabelError) / 4.

% Query example:
% ?- measurement_risk(0.42, 0.12, 0.24, 0.08, Score).

File: articles/proxy-variables-and-measurement-error/cpp/proxy_measurement_risk.cpp
#include <iostream>

int main() {
    double validity_gap = 0.42;
    double missingness = 0.12;
    double differential_error = 0.24;
    double label_error = 0.08;
    double score = (validity_gap + missingness + differential_error + label_error) / 4.0;
    std::cout << "measurement_risk_score=" << score << "\n";
    return 0;
}
